src/jit/namedintrinsiclist.h

```diff
@@ -20,6 +20,10 @@ enum NamedIntrinsic : unsigned int
 #define HARDWARE_INTRINSIC(id, name, isa, ival, size, numarg, t1, t2, t3, t4, t5, t6, t7, t8, t9, t10, category, flag) \
     NI_##id,
 #include "hwintrinsiclistxarch.h"
+    NI_SW_INTRINSIC_START,
```

Comment:
We should have `NI_HW_INTRINSIC_END` before this, to clearly define the separate ranges.

Reply:
Agrred, I have ended up with same conclusion above 😄